student.rb

```diff
@@ -1,6 +1,10 @@
 require './person'
+require './classroom'
 
```

Comment:
- [ ] You are currently missing the Book class please add it and all the associations

Reply:
@hiromataba the book class is added to the person. Could you please show the connection?

Reply:
- [ ] @V0pe  Where is your book class please ?

Reply:
@hiromataba the book class is linked straight to the person class. The student inherited the add_rentals from the person class